File: Zeze/Arch/LocalRemoveEventArgument.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Zeze.Builtin.Online;

namespace Zeze.Arch
{
    public class LocalRemoveEventArgument : EventArgs
    {
        public string Account { get; set; }
        public string ClientId { get; set; }
        public BLocal LocalData { get; set; }
    }
}

File: ZezeJava/ZezeJava/src/main/java/Zeze/Arch/LocalRemoveEventArgument.java
package Zeze.Arch;

import Zeze.Builtin.Online.BLocal;
import Zeze.Util.EventDispatcher;
import org.jetbrains.annotations.NotNull;

public class LocalRemoveEventArgument implements EventDispatcher.EventArgument {
	public final @NotNull String account;
	public final @NotNull String clientId;
	public final @NotNull BLocal local;

	public LocalRemoveEventArgument(@NotNull String account, @NotNull String clientId, @NotNull BLocal localData) {
		this.account = account;
		this.clientId = clientId;
		this.local = localData;
	}
}
